src/Worker/Notifier.php

```diff
@@ -502,7 +502,7 @@ private static function delivery(string $cmd, int $post_uriid, int $sender_uid,
 		$a = DI::app();
 		$delivery_queue_count = 0;
 
-		if ($target_item['verb'] == Activity::ANNOUNCE) {
+		if (!empty($target_item['verb']) && ($target_item['verb'] == Activity::ANNOUNCE)) {
```

Comment:
I think the extra `!empty()` is superfluous here as `Activity::ANNOUNCE` is a non-empty string by itself.

Reply:
`$target_item['verb']` can be empty under certain circumstances as I learned from the logs.